### ISSUE TITLE
Ensure color is not the sole means of communicating information

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#53]
## Description
When colors are used as the sole method for identifying screen elements, controls, or giving instructions, persons who are blind, color blind, or have low vision may find the web page unusable. This provision does not prohibit the use of color to enhance the identification of important features. It does, however, require that some other method of identification, such as on-screen text, be combined with the use of color.

Color must not be the sole method of indicating a link within non-linked text.

### RTE content Links (Standard Links)

- [x] Standard teal links in RTE content should include an underline

![cec0d09547a1a94aeab90237f015b1a31253ef53-1641304980](https://user-images.githubusercontent.com/8166060/152437248-91f219d2-650b-42c4-a6e7-fa6e0b90da2d.png)

### SlideBlock

- [x] Mobile Slide Carousel controls should use at least one other visual indicator for active slide status

![4b511f45114f89d00b6cf674d75480bb17261766-1641296891](https://user-images.githubusercontent.com/8166060/152437302-d303e96e-960f-4314-ad98-6cf27f9b1498.png)

### CTA Grid, Related Content, News Grid, Jobs Board, Gallery Grid

- [x]  Grid Links ("Cards") should have second visual indicator on focus and hover in addition to color change (e.g. outline, scale, shape change, etc.).

![280d8e1772e5ea80edd22fca33e51e42532e8832-1640949634](https://user-images.githubusercontent.com/8166060/152437351-2caae9a4-6007-4307-8549-abebadc7dcd4.png)

**New Design**
[This is a dropbox link with updated design for Grid Items focus & hover states](https://www.dropbox.com/s/fd70j7zcwoz54ci/20220223%20CTA%20Grid.pdf?dl=0)

### Breadcrumbs

- [x] Breadcrumb links should have underline
- [x] "Current Page" breadcrumb link should have additional visual indicator to set it apart from "inactive" breadcrumb links

![40c03503e9444e8efef881e023552b362bf19970-1641199162](https://user-images.githubusercontent.com/8166060/152437428-2d248aac-cac0-4898-9db1-97e3cf5dfc76.png)

**New Design**
[This is a dropbox link with updated design for breadcrumb links](https://www.dropbox.com/s/it2m5udde6hdyay/20220325%20Breadcrumb%20styles.pdf?dl=0)

### Pagination Links

- [x] Pagination Links should have underline

<img width="450" src="https://user-images.githubusercontent.com/8166060/152450785-c16e575e-69cb-481f-bd62-cd54a07c3b8c.png">

## CIC Estimate
1 point

**Comment by blnkt** [comment]
For: **CTA Grid, Related Content, News Grid, Jobs Board, Gallery Grid**

[This is a dropbox link with updated design for Grid Items focus & hover states](https://www.dropbox.com/s/fd70j7zcwoz54ci/20220223%20CTA%20Grid.pdf?dl=0)

**Comment by blnkt** [comment]
For: **SlideBlock**

[This is a dropbox link with updated design for mobile SlideBlock controls active slide styles.](https://www.dropbox.com/s/kblm682cwojewk1/20220223%20Mobile%20slide%20block.pdf?dl=0)

**Comment by blnkt** [comment]
This issue is no longer blocked by design.  All new designs are provided through dropbox links.
